README.md

```diff
@@ -100,14 +99,14 @@ Before deploying the python code to your CPX device, you need to format your dev
 
 1. Download the firmware with the .uf2 file (link: https://learn.adafruit.com/adafruit-circuit-playground-express/circuitpython-quickstart)
 2. Download the lastest version of the cpx library (link: https://learn.adafruit.com/welcome-to-circuitpython/circuitpython-libraries).  
-   **_Note:_** Make sure you name your file main.py or code.py: the device automatically runs the first file that is likely named.
+   **_Note:_** Make sure you name your file `main.py` or `code.py`: the device automatically runs the first file that is likely named.

```

Comment:
Maybe we can point out the reason for that? E.g. [this reference](https://learn.adafruit.com/welcome-to-circuitpython/creating-and-editing-code#naming-your-program-file-7-32) or explain that this is the convention for CircuitPython.

Reply:
Will update this in next commit